tests/unit/conftest.py

```diff
@@ -55,17 +55,17 @@ def inner(reg: Optional[str] = None) -> Path:
 
 
 @pytest.fixture
-def cycling_mode(monkeypatch):
-    """Set the Cylc cycling mode and return its value."""
-    def _cycling_mode(integer=True):
-        mode = INTEGER_CYCLING_TYPE if integer else ISO8601_CYCLING_TYPE
+def cycling_type(monkeypatch):
+    """Set the Cylc cycling type and return its value."""
+    def _cycling_type(integer=True):
+        ctype = INTEGER_CYCLING_TYPE if integer else ISO8601_CYCLING_TYPE
```

Comment:
I hadn't realised the difference between cycling type and mode. If I understand correctly:
- Type is either `integer` or `iso8601`
- Mode is either `integer`, `gregorian`, `360day`, `365day` etc

Btw, just a heads up this is going to conflict with my PR #4213 - https://github.com/cylc/cylc-flow/pull/4213/files#diff-0cae8a7ee2d37098b1ad84b543d17cfc1e8535eed5fd6abac88c668bfe354cbb

Reply:
That's correct, I had forgotten the distinction myself.

Reply:
> Btw, just a heads up this is going to conflict with my PR

And unfortunately (for me!) I just merged that PR :grimacing:

Reply:
Deconflicted.  And I've attempted to straighten out the cycling type vs mode stuff in the process.